### PR TITLE
Add --json mode

### PR DESCRIPTION
**Pull request by LiteracyFanatic**

### CODE REVIEW

**Comment by LiteracyFanatic** [comment]
So it looks like this fails to build on the LTS Ubuntu machine. If I understand correctly, it just needs some extra includes that aren't necessary with more recent versions of Qt correct?

File: src/cli.cpp [hunk 1]
```diff
@@ -87,6 +90,12 @@ void Cli::process(const QCoreApplication &app)
         parser.showHelp();
     }
 
+    if (parser.isSet(json) && (parser.isSet(audioOnly) || parser.isSet(brief)))
+    {
```
Comment (Shatur95) [inline src/cli.cpp:94]:
```suggestion
    if (parser.isSet(json) && (parser.isSet(audioOnly) || parser.isSet(brief))) {
```

File: cmake/ExternalLibraries.cmake [hunk 2]
```diff
@@ -19,7 +19,7 @@ FetchContent_Declare(QHotkey
 
 FetchContent_Declare(QOnlineTranslator
     GIT_REPOSITORY https://github.com/crow-translate/QOnlineTranslator
-    GIT_TAG 1.4.0
+    GIT_TAG 64e3e8876d571cb7bd6a5b0e71dd73b1aaf213a0
```
Comment (Shatur95) [inline cmake/ExternalLibraries.cmake:22]: I just fixed missing header in `QOption` for Ubuntu to fix CI. Could you update to `297befe8414b6b49f27b0b12532bfd71ab7ecf5b`?

**Review by Shatur95 — CHANGES_REQUESTED** [review]
Thank you!

File: cmake/ExternalLibraries.cmake (hunk 2 above)
Comment (LiteracyFanatic) [reply cmake/ExternalLibraries.cmake]: Done.

**Review by Shatur95 — APPROVED** [review]

**Comment by Shatur95** [comment]
Thanks for working on it!

**Comment by Shatur95** [comment]
One more thing: I removed `errorString` value because Crow will stop working before on error an return non-zero error code. So, just check the execution result before parsing JSON output.

**Comment by LiteracyFanatic** [comment]
> Thanks for working on it!

Sure thing, it was a good learning experience.

> One more thing: I removed `errorString` value because Crow will stop working before on error an return non-zero error code. So, just check the execution result before parsing JSON output.

Makes sense.

**Pull request merged**